packages/teams/src/utils/get-allowed-group-access.ts

```diff
@@ -10,10 +11,15 @@ import { AGOAccess } from "../types";
  */
 export function getAllowedGroupAccess(
   requestedAccess: AGOAccess,
-  user: IUser
+  user: IUser,
+  portal: IPortal
```

Comment:
This should be optional so that it's not a breaking change:

```suggestion
  portal?: IPortal
```